Makefile

```diff
@@ -48,11 +50,22 @@ tarballs:
 docker:
 	@docker build -t prometheus:$(shell git rev-parse --short HEAD) .
 
+rpm-nosystemd:
+	@echo ">> building rpm package for no-systemd distros"
+	@rpmbuild --buildroot "$(PWD)/.build/nosystemd" -D "_topdir $(PWD)" -D "_builddir $(PWD)/.build/nosystemd" -D "src_root $(PWD)" -D "version $(VERSION)" -D 'use_systemd 0' -bb prometheus.spec
+	@mv RPMS/*/*.rpm "$(PWD)"/
+
+rpm-systemd:
+	@echo ">> building rpm package for systemd distros"
+	@rpmbuild --buildroot "$(PWD)/.build/systemd" -D "_topdir $(PWD)" -D "_builddir $(PWD)/.build/systemd" -D "src_root $(PWD)" -D "version $(VERSION)" -D 'use_systemd 1' -bb prometheus.spec
+	@mv RPMS/*/*.rpm "$(PWD)"/
+
+rpm:	build rpm-systemd rpm-nosystemd
```

Comment:
Can you not produce a single RPM? I imagine there's more dimensions that just systemd and a combinatorial explosion is unwise. It'd be best to only provide a standard init script, as that's going to work everywhere.